tasks/OracleLinux

```diff
@@ -0,0 +1 @@
+CentOS
```

Comment:
Wow, this works?

Reply:
Yes. We have one host that is OracleLinux we have to install a few things for some of the plugins install: 

    - "gcc-c++"
    - "kernel-devel"
    - "make"